src/java.desktop/share/classes/javax/swing/plaf/synth/SynthTabbedPaneUI.java

```diff
@@ -153,9 +146,6 @@ private boolean scrollableTabLayoutEnabled() {
      */
     @Override
     protected void installDefaults() {
-        selectColor = UIManager.getColor("TabbedPane.selected");
-        contentOpaque = UIManager.getBoolean("TabbedPane.contentOpaque");
-        unselectedBackground = UIManager.getColor("TabbedPane.unselectedBackground");
         updateStyle(tabPane);
```

Comment:
guess you missed adding this back
tabsOpaque = UIManager.getBoolean("TabbedPane.tabsOpaque");

Reply:
ok..I guess I am wrong..

Reply:
Not actually... `tabsOpaque` property was added in JDK-8226990 but later on it it is moved to SynthPainterImpl and GTKPainter classes under [JDK-8335130](https://bugs.openjdk.org/browse/JDK-8335130) issue.